src/S3/S3Transfer/Models/DownloadDirectoryRequest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Aws\S3\S3Transfer\Models;
+
+use Aws\Arn\ArnParser;
+use Aws\S3\S3Transfer\Progress\TransferListener;
+use InvalidArgumentException;
+
+final class DownloadDirectoryRequest extends TransferRequest
+{
+    public static array $configKeys = [
+        's3_prefix' => 'string',
+        'filter' => 'callable',
+        'download_object_request_modifier' => 'callable',
+        'failure_policy' => 'callable',
+        'max_concurrency' => 'int',
+        'track_progress' => 'bool',
+        'target_part_size_bytes' => 'int',
+        'list_objects_v2_args' => 'array',
+        'fails_when_destination_exists' => 'bool',
+    ];
+    public const DEFAULT_MAX_CONCURRENCY = 100;
```

Comment:
How did we land on 100? V1 has a default of 5

Reply:
It is the default value set in the technical specs.